src/Sema.zig

```diff
@@ -6924,6 +7010,24 @@ fn validateSwitchItemEnum(
     return sema.validateSwitchDupe(block, maybe_prev_src, switch_prong_src, src_node_offset);
 }
 
+fn validateSwitchItemError(
+    sema: *Sema,
+    block: *Block,
+    seen_errors: *SwitchErrorSet,
+    item_ref: Zir.Inst.Ref,
+    src_node_offset: i32,
+    switch_prong_src: Module.SwitchProngSrc,
+) CompileError!void {
+    const item_tv = try sema.resolveSwitchItemVal(block, item_ref, src_node_offset, switch_prong_src, .none);
+    // TODO: Do i need to typecheck here?
```

Comment:
I don't see any missing code here. I think it's fine to merge as-is - I'm sure we'll find out sooner or later if additional logic is needed, and we have #363 to scoop up this todo item later :)